src/styles/stars.scss

```diff
@@ -0,0 +1,29 @@
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 16px;
+  width: 200px;
+}
+
+.card__stars-list {
+  display: flex;
+}
+
+.card__star:not(:last-child) {
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
```

Comment:
The border-radius value seems unusually small. If the intention is to create a rounded star, consider increasing the border-radius or removing it if it's not needed.